node/readline.ts

```diff
@@ -1,15 +0,0 @@
-import {
-  clearLine,
-  clearScreenDown,
-  cursorTo,
-  moveCursor,
-} from "./internal/readline/callbacks.js";
-import { emitKeypressEvents } from "./internal/readline/emitKeypressEvents.js";
-
-export default {
-  clearLine,
-  clearScreenDown,
-  cursorTo,
-  moveCursor,
```

Comment:
This module should be kept and reorganized in similar fashion to `buffer.ts` and `events.ts`

Reply:
Done